server/routes/api/users.ts

```diff
@@ -188,7 +189,7 @@ router.post("users.update", auth(), async (ctx) => {
   }
   if (preferences) {
     assertKeysIn(preferences, UserPreference);
-    if (preferences.rememberLastPath) {
+    if (has(preferences, UserPreference.RememberLastPath)) {
```

Comment:
We might want to move this `UserPreference` enum to `/shared/` it could be used in some of the frontend components/types then.